frontend/stubs/list_methods.py

```diff
@@ -0,0 +1,52 @@
+"""Stub file for methods invoked on lists"""
+from typing import TypeVar, List
+
+T = TypeVar("T")
+
+
+def append(l: List[T], e: T) -> None:
```

Comment:
Okay I'll add it in PR #27, because it will require to have a comment in the code indicating that it throws an exception, and such mechanism for special comments parsing is already implemented in #27